easybuild/easyconfigs/h/HEASoft/HEASoft-6.27.2-GCCcore-4.9.3.eb

```diff
@@ -0,0 +1,89 @@
+name = 'HEASoft'
+version = '6.27.2'
+
+
+dependencies = [
+    ( 'CFITSIO', '3.48' ),
+    ( 'Perl', '5.30.2' ),
+]
+
+easyblock = 'ConfigureMake'
+
+homepage = ''
+description = """"""
+
+toolchain = {'name': 'GCCcore', 'version': '4.9.3'}
```

Comment:
You are specifying a very old compiler here, but the dependencies you specify (Perl 5.30.2) only exists for much newer GCCcore.  Unless you have backported a whole lot of easyconfigs to this really old version, it will not work as is (and it will definitely not work in this repository. 
Version 9.3.0 should work fine though (matches with your CFITSIO and Perl dependencies).

Reply:
I did back-port about easyconfigs. I had some trouble building, this software can be delicate.
Also I was trying to extrapolate from whatever where was already available on our cluster. 
I will try with 9.3.0, it will take some time.

Reply:
moved to 9.3.0, works fine. Thanks for the suggestion!